docs/user/editor/patterns.md

```diff
@@ -86,6 +86,16 @@ Individual timings for each pattern stop can also be assigned either through exp
 - Travel time – the time it took to get from the previous station (should be 00:00 on the first stop of the sequence)
 - Dwell time – the time the vehicle rests at the stop
 
+## Tutorial Video
```

Comment:
Same remark/suggestion regarding moving this section to the top?
Also, some text wraps around the sides of the video:

![image](https://user-images.githubusercontent.com/56846598/88347186-30365e00-cd18-11ea-957c-c11099f93672.png)

Reply:
Never mind moving this, I realized there were more!

Reply:
Suggestion: But then it would make sense to reiterate the title of the section/video in the header, for instance, "Tutorial Video: Creating Trip Patterns".

Reply:
I agree with the title of the section in the header. I can't seem to replicate the text wrapping issue however.

Reply:
Same deal. In `patterns.md`, insert a blank line between the `<iframe>` and the preceding paragraph.